packages/core/src/schema/models/evaluationResults.ts

```diff
@@ -0,0 +1,51 @@
+import { relations } from 'drizzle-orm'
+import { bigint, bigserial, index, text } from 'drizzle-orm/pg-core'
+
+import { documentLogs, latitudeSchema, providerLogs } from '..'
+import { timestamps } from '../schemaHelpers'
+import { evaluations } from './evaluations'
+
+export const evaluationResults = latitudeSchema.table(
+  'evaluation_results',
+  {
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
+    evaluationId: bigint('evaluation_id', { mode: 'number' })
+      .notNull()
+      .references(() => evaluations.id, { onDelete: 'cascade' }),
+    documentLogId: bigint('document_log_id', { mode: 'number' })
+      .notNull()
+      .references(() => documentLogs.id),
+    providerLogId: bigint('provider_log_id', { mode: 'number' })
+      .notNull()
+      .references(() => providerLogs.id),
+    result: text('result').notNull(),
+    ...timestamps(),
+  },
+  (table) => ({
+    evaluationIdx: index('evaluation_idx').on(table.evaluationId),
+    evaluationResultDocumentLogIdx: index('document_log_idx').on(
+      table.documentLogId,
+    ),
+    evaluationResultProviderLogIdx: index('provider_log_idx').on(
+      table.providerLogId,
+    ),
+  }),
+)
```

Comment:
Yes, I am not sure if adding different types of result fields (boolean, numeric, text,...) or even different results table or maybe not maybe a unique text field as you have here is enough 

🧠 Think what we miss if we have every kind of possible result store as text

Reply:
Talked with product. We're going to store only text for now. We're not going to perform aggregations other than group counts, so that's fine. Will have to revisit this in the future though.

Reply:
Hmm I think it's fine for now? My only concern is other types of evaluations that do not involve prompting an llm for evaluation results (like user submitted evaluations) which suggest a polymorphic approach instead. Bring it up with product to see if this is something we wanna support in the short to mid term and if so then I would consider using polymorphism from the get go.

Reply:
Talked with product. Not something we're going to be adding in short/mid-term. All evaluations will be performed with LLMs for now, even dumber ones.